testing/test_pytest_html.py

```diff
@@ -418,6 +418,39 @@ def test_fail():
             assert link in html
             assert os.path.exists(src)
 
+    @pytest.mark.parametrize('src_type', ["https://", "file://"])
+    def test_extra_image_non_b64(self, testdir, src_type):
+        content = src_type
+        testdir.makeconftest("""
+            import pytest
+            @pytest.mark.hookwrapper
+            def pytest_runtest_makereport(item, call):
+                outcome = yield
+                report = outcome.get_result()
+                if report.when == 'call':
+                    from pytest_html import extras
+                    report.extra = [extras.image('{0}')]
+        """.format(content))
+        testdir.makepyfile('def test_pass(): pass')
+        result, html = run(testdir, 'report.html')
+        assert result.ret == 0
+        assert '<img src="{0}"/>'.format(content) in html
+
+    def test_extra_text_encoding(self, testdir):
```

Comment:
I've started looking at the image/slider PR @davehunt, I've got most of your requests in place, I just need to fine tune the CSS. 

However, I'd prefer to get this PR out of the way first, since they hit more or less exactly the same code.